README.md

```diff
@@ -150,27 +142,10 @@ Congratulations, you’ve just built your first Kubernetes charm using `ops`!
 
 ### Clean up
 
-> See [Charm SDK | Clean up](https://juju.is/docs/sdk/dev-setup#heading--automatic-set-up-an-ubuntu-charm-dev-vm-with-multipass).
+> See [Juju | Tear things down](https://canonical-juju.readthedocs-hosted.com/en/latest/user/howto/manage-your-deployment/manage-your-deployment-environment/#tear-things-down). <br> Choose the automatic track.
 
 ## Next steps
 
-### Learn more
-- Read our [user documentation](https://juju.is/docs/sdk/ops), which includes other guides showing `ops` in action
-- Dig into the [`ops` API reference](https://ops.readthedocs.io/en/latest/)
-
-### Chat with us
-
-Read our [Code of conduct](https://ubuntu.com/community/code-of-conduct) and:
-- Join our chat: [Mattermost](https://chat.charmhub.io/charmhub/channels/ops)
-- Join our forum: [Discourse](https://discourse.charmhub.io/)
-
-### File an issue
```

Comment:
What's the reason for removing these links? Obviously the doc one needs to be collapsed into the first point rather than pointing to Juju (which probably hasn't been the thing to do for some time...) but I think there's value in having this. If you're reading this on GitHub it's somewhat obvious that there is an issues tab (but it doesn't hurt to be explicit), and on PyPI there is an "issues" link but this spells it out more clearly, and if you only have the dist, then the is the only link in the readme to GitHub I think.

Reply:
I think these are moved to the bottom, 3 links on one line; so not totally removed.

Reply:
All this info is now given more compatible direct after Next steps. We don't have as many links anymore. (E.g., there's no separate docs vs. API reference.